modules/nw-dhcp-configuration-guide.adoc

```diff
@@ -0,0 +1,42 @@
+[id="dhcp-configuration-guide_{context}"]
+= Creating a DHCP server
+
+A DHCP server can be created if there is not an existing DHCP server available to you to use for creating DHCP.
```

Comment:
I would delete the last part of this sentence... "~~to you to use for creating DHCP~~."